### ISSUE TITLE
macOS Catalina [bug]

### ISSUE THREAD

**Issue by JoNilsson** [OJFord/passbase#16]
After Catalina update it seems there may be some breaking changes related to system access / permissions to the KBFS.  

`.passbase` dir does certainly exist. 

```sh
→ passbase ls                                                                                                                           19-10-10 / 10:48 (/bin/zsh)
Passbase directory does not exist in KBFS, creating...
thread 'main' panicked at 'Failed to create Passbase directory: Error { repr: Os { code: 2, message: "No such file or directory" } }', /Users/rustbuild/src/rust-buildbot/slave/nightly-dist-rustc-mac/build/src/libcore/result.rs:870
note: Run with `RUST_BACKTRACE=1` for a backtrace.
```

**Comment by OJFord** [comment]
Thanks for the report - I don't have a Catalina system to test with at the moment - does it reproduce repeatedly, or if you `passbase ls` again does it 'wake up'? What about if you access the directory from Keybase app first?

**Comment by JoNilsson** [comment]
> does it reproduce repeatedly, or if you passbase ls again does it 'wake up'? 

Yes.   Any function that calls to the KBFS fails.  The functionality doesn't *"wake up"*
ie:  
```sh
passbase create catalina.test -n 8                                                                                                                                                                                                                 101 - 19-10-11 / 9:39 (/bin/zsh)
Passbase directory does not exist in KBFS, creating...
thread 'main' panicked at 'Failed to create Passbase directory: Error { repr: Os { code: 2, message: "No such file or directory" } }', /Users/rustbuild/src/rust-buildbot/slave/nightly-dist-rustc-mac/build/src/libcore/result.rs:870
```

the `create` will repeatedly fail.  as does `ls`

 > What about if you access the directory from Keybase app first?

Nope.  Tried again,  on the edge Keybase Version 4.6.0-2019 that was pushed out earlier this week.  
Same result after opening the `.passbase` dir via UI / Finder.

**Comment by OJFord** [comment]
Interesting, thanks. I'll update myself and have a look, don't really have any other suggestion without being able to fiddle with it.

**Comment by JoNilsson** [comment]
I was hesitant, but so far Catalina hasn't given me much grief. 

I was looking in `main.rs` [here](https://github.com/OJFord/passbase/blob/master/src/main.rs#L113):
```rs
let passbase_dir = Path::new("/keybase/private")
```

maybe this should actually be ```/volumes/Keybase/private/``` ??

**Comment by JoNilsson** [comment]
put in a PR that I think fixes this issue (i hope)  #17
